falco/generated/helm-values.md

```diff
@@ -1,5 +1,5 @@
 # Configuration values for falco chart
-`Chart version: v2.0.3`
+`Chart version: v2.0.5`
```

Comment:
You are bumping to `2.0.6` in `Chart.yaml`. Why here only to `2.0.5`? (Also thanks for adding documentation for my introduced flag `mounts,enforceProcMount` which I forgot in my previous PR.)

Reply:
Nice catch! Thank you! 
That because i run `make docs` before bumping chart version to `v2.0.6`! I think we should add a check in the CI which makes sure that the `make docs` have been executed.

Reply:
Nice catch! Thank you!
That because i run make docs before bumping chart version to v2.0.6! I think we should add a check in the CI which makes sure that the make docs have been executed.